srcpkgs/oneko/patches/rmshape_fixtypos.diff

```diff
@@ -0,0 +1,69 @@
+diff --git a/oneko.c b/oneko.c
+index d2b81fe..39b8b1d 100644
+--- a/oneko.c
++++ b/oneko.c
+@@ -618,13 +618,11 @@ InitScreen(DisplayName)
+     XSynchronize(theDisplay,True);
+   }
+ 
+-#ifdef SHAPE
+   if (!NoShape && XShapeQueryExtension(theDisplay,
+ 				       &event_base, &error_base) == False) {
+-    fprintf(stderr, "Display not suported shape extension.\n");
++    fprintf(stderr, "Display does not support shape extension.\n");
+     NoShape = True;
+ 				       }
+-#endif SHAPE
+ 
+   theScreen = DefaultScreen(theDisplay);
+   theDepth = DefaultDepth(theDisplay, theScreen);
+@@ -803,13 +801,11 @@ DrawNeko(x, y, DrawAnime)
+       theChanges.x = x;
+       theChanges.y = y;
+       XConfigureWindow(theDisplay, theWindow, CWX | CWY, &theChanges);
+-#ifdef SHAPE
+       if (NoShape == False) {
+ 	XShapeCombineMask(theDisplay, theWindow, ShapeBounding,
+ 			  0, 0, DrawMask, ShapeSet);
+ 
+       }
+-#endif SHAPE
+       if (DontMapped) {
+ 	XMapWindow(theDisplay, theWindow);
+ 	DontMapped = 0;
+diff --git a/oneko.man b/oneko.man
+index d6d2b40..f24f19c 100644
+--- a/oneko.man
++++ b/oneko.man
+@@ -1,6 +1,6 @@
+ .TH ONEKO 6
+-.SH NAME oneko
+-The program oneko creates a cute cat chasing around your mouse cursor.
++.SH NAME
++oneko \- Creates a cute cat chasing around your mouse cursor.
+ .SH SYNOPSIS
+ .B oneko
+ [\fB-help\fP] [\fB-tora\fP]
+@@ -22,8 +22,7 @@ start sleeping.
+ Prints help message on usage.
+ .TP
+ .B -tora
+-Make cat into "tora-neko".  "Tora-neko" means cat wite tiger-like stripe.
+-I don't know how to say it in English.
++Make cat into "tora-neko". a cat with a tiger-like stripe.
```

Comment:
that first period seems like it should be a coma

Reply:
yeah i forgot. i simply removed the sentence after it. thanks for pointing it out!